requirements.txt

```diff
@@ -4,8 +4,6 @@ autopep8==1.4.3
 certifi==2018.11.29
 cffi==1.11.5
 chardet==3.0.4
-coreapi==2.3.3
```

Comment:
@BarnaTB Please add the packages you removed here , some are required for the api documentation.